test/gtest/uct/uct_test.h

```diff
@@ -478,7 +479,11 @@ class test_uct_iface_attrs : public uct_test {
     rc_verbs,           \
     dc_mlx5,            \
     ud_verbs,           \
-    ud_mlx5,            \
+    ud_mlx5
+
+
+#define UCT_TEST_IB_AND_GGA_TLS \
```

Comment:
Q: maybe add `gga_mlx5` to the  list of TLS and skip when it's not available?

Reply:
init/cleanup also takes time